tests/migrate/test_concat.py

```diff
@@ -157,21 +153,20 @@ def setUp(self):
             <!ENTITY channel.description.end    " channel. ">
         ''')
 
-    @unittest.skip('Parser/Serializer trim whitespace')
     def test_concat_replace(self):
         msg = FTL.Message(
             FTL.Identifier('channel-desc'),
             value=CONCAT(
                 COPY(self.strings, 'channel.description.start'),
-                EXTERNAL_ARGUMENT('channelname'),
+                FTL.Placeable(EXTERNAL_ARGUMENT('channelname')),
                 COPY(self.strings, 'channel.description.end'),
             )
         )
 
         self.assertEqual(
             evaluate(self, msg).to_json(),
             ftl_message_to_json('''
-                channel-desc = You are on the { $channelname } channel.
+                channel-desc = You are on the { $channelname } channel.\u0020
```

Comment:
This smells like anticipating a particular solution to [bug 1374246](https://bugzilla.mozilla.org/show_bug.cgi?id=1374246). Want to remove the trailing whitespace in line 153 instead?

Reply:
Good point.